src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,20 @@
 package mate.academy.service;
 
 public class AuthenticationService {
+    private static final UserService userService = new UserService();
+
     /**
      * Imagine that some user wants to login to your site.
      * You should check if user credentials (login and password) are valid or not.
      * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
+     *
+     * @param email    - user's email
      * @param password - user's password
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        return email.equals(userService.findByEmail(email).getEmail())
```

Comment:
It's better to create User local variable
`User user = userService.findByEmail(email);`
and then check if user != null and have equals password 

In such approach, you won't call userService.findByEmail(email) twice and won't get NullPointerException if User is null